webscalesqlclient/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ cmake_minimum_required(VERSION 2.8.0)
 include(ExternalProject)
 ExternalProject_Add(
   webscalesqlclient
+  URL file://${CMAKE_CURRENT_SOURCE_DIR}/webscalesql-5.6
```

Comment:
Sure, supporting out-of-tree builds is fine. But this breaks in-tree builds. I can't merge it until that is fixed.